roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Ensure Keycloak host config directory
+  file:
+    state: directory
+    path: "{{ keycloak_host_config_dir }}"
+    owner: root
+    mode: 0644
+  become: true
+
+- name: Template Keycloak cache ISPN configuration
+  template:
+    src: "{{ keycloak_cache_config_file_template }}"
+    dest: "{{ keycloak_host_config_dir }}/{{ keycloak_cache_config_file }}"
+    owner: root
+    mode: 0644
```

Comment:
Stricter permissions required here - it has the DB password in

Reply:
Nothing actually gets exposed in this file (which is quite nice!). There's functionality in the container for filling in `"${env.KC_DB_PASSWORD}"` in the xml file with an environment variable at runtime.

Reply:
Ack, makes sense.